docs/deploying-on-k8s.md

```diff
@@ -36,8 +36,8 @@ local crng = import 'carbon-relay-ng/crng.libsonnet';
   crng: crng {
     _config+:: {
       namespace: 'mynamespace',
-      crng_route_host: 'https://tsdb-1-<instance name>.hosted-metrics.grafana.net/metrics',
```

Comment:
I updated this value as the cloud graphite v5 endpoints don't follow the `tsdb-1-<instance name>.hosted-metrics.grafana.net` pattern.